pom.xml

```diff
@@ -268,5 +274,32 @@
       </plugins>
     </pluginManagement>
   </build>
-
+  <profiles>
+    <profile>
+      <id>project-repositories</id>
```

Comment:
no need to add this since it's already declared in parent POM and thus will be inherited, see : 
https://github.com/Meeds-io/maven-parent-pom/blob/develop/pom.xml#L1894

Reply:
It is necessary as maven-parent-pom is in our public repository. The github action is unable to find it if we dont put the repository here